assets/js/util/insufficient-permissions-error-description.test.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * Tests for getInsufficientPermissionsErrorDescription.
+ *
+ * Site Kit by Google, Copyright 2021 Google LL
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { getInsufficientPermissionsErrorDescription as createErrorMessage } from './insufficient-permissions-error-description';
+
+describe( 'getInsufficientPermissionsErrorDescription', () => {
+	describe( 'default parameters', () => {
```

Comment:
I think there is no need in these nested describe blocks. Let's have one level of depth for all these tests. Usually, we try to avoid using nested describe blocks with a single test.

Reply:
Thanks for the suggestion, let me update this one.